chapters/annotations.tex

```diff
@@ -460,7 +460,9 @@ \section{Annotations for Code Generation}\label{annotations-for-code-generation}
 \end{semantics}
 \end{annotationdefinition}
 
-\section{Annotations for Simulation Experiments}\label{annotations-for-simulation-experiments}
+\section{Annotations for Simulations}\label{annotations-for-simulations}
+These annotations define how models can be simulated.
```

Comment:
Did we say that a test case could be more general than something that would be possible to simulate even if errors were corrected?  For example, couldn't a partial model be a test case that demonstrates some illegal use?

Reply:
I changed that to be more general.

Reply:
Yes, the new introduction is a better description of the two subsections, but maybe the two annotations are simply too different to fit under the same section?

Reply:
I still think they are related. However, I think we should more do a pass to make that chapter more consistent; as sections vary wildly in length.

Reply:
OK, seems like we've had enough of discussion for now then.